frontend/package.json

```diff
@@ -77,6 +77,14 @@
     ]
   },
   "dependencies": {
+    "@apollo/client": "^3.8.5",
+    "@chakra-ui/react": "^2.8.1",
+    "@diamondlightsource/ui-components": "^1.0.2",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "framer-motion": "^10.16.4",
```

Comment:
We should work out what's going on with the `@diamondlightsource/ui-components` dependencies before merging this